templates/home.html

```diff
@@ -0,0 +1,28 @@
+<html lang="en">
+  <head>
+    <style>
+      .button {
+        background-color: #000000;
+        border: none;
+        color: white;
+        padding: 20px 34px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 20px;
+        margin: 4px 2px;
+        cursor: pointer;
+      }
+      .button:hover {
+      background-color: #FFFFFF;
+      color: black;
+      }
+    </style>
+  </head>
+  <body style="background-color:powderblue;">
+    <h1>Demo Website Home Page</h1>
+    <a href="http://localhost:5000/register" class="button">Sign Up</a>
```

Comment:
Can we pass this url instead of manually coding them here. Or explore how url_for can be used.

Reply:
Yes I agree that somehow passing the url makes all of this html code much more ready for change (e.g. a change in hostname, port, etc.)

I've done this through using the flask url_for method like you mentioned.  That was a really good suggestion